client/src/components/ManageAccount/tests/ManageAccountModals.test.js

```diff
@@ -362,37 +399,25 @@ describe('Edit Required Information', () => {
       }
     };
     const storeWithFields = mockStore({ profile: stateWithFields });
+    stateWithFields['data']['demographics']['phone'] = '122'
+    stateWithFields['data']['demographics']['email'] = 'email';
 
     rerender(
       <Provider store={storeWithFields}>
         <EditRequiredInformationModal />
       </Provider>
     );
-
-    const emailField = getByLabelText(/email/);
-    const phoneField = getByLabelText(/phone/);
     const submitButton = getByLabelText(/required-submit/);
-
-    // Invalid Entries
-    fireEvent.change(emailField, {
-      target: {
-        value: 'email'
-      }
-    });
-    fireEvent.change(phoneField, {
-      target: {
-        value: '123'
-      }
-    });
-    const clickSpy = () => jest.fn();
-    submitButton.addEventListener('click', clickSpy, false);
+    const clickSpy = jest.spyOn(submitButton, 'click');
     fireEvent.click(submitButton);
-    wait(() => {
-      expect(getByText(/Please enter a valid email address/)).toBeDefined();
-      expect(getByText(/Phone number is not valid/)).toBeDefined();
+
+    await wait(() => {
+      expect(getByText('Phone number is not valid')).not.toBeNull();
+      expect(getByText('Please enter a valid email address')).toBeDefined();
```

Comment:
Why the difference in tests here?

Reply:
@rstijerina I think I introduced this discrepancy. I was seeing some inconsistencies when using `.toBeDefined();` This is because null is not strictly equal to undefined, but can appear that way depending on how the two are compared. As in:

```
null === undefined // false
null == undefined // true
```

So value `null` can be interpreted as defined. We may want to just use `.not.toBeNull();` here since I believe that's what we get if we get a miss on `getByText`

Reply:
@rstijerina this was an existing test for "should show users errors if the fields are missing or invalid" that needs to be updated because it didn't work.

Reply:
changed to `getByText` and `toBeDefined `for both of test since this test want to get an error message to be defiend.